KarhooUISDK/Models/LoyalityMode.swift

```diff
@@ -33,7 +33,7 @@ public enum KarhooLoyaltyError: KarhooError {
     }
 }
 
-public enum LoyaltyMode {
+public enum LoyaltyMode: Equatable {
```

Comment:
It's not required to make the fix work

Reply:
It is needed for test targets

Reply:
Maybe add a comment inside the code so that we don't lose this piece of information

Reply:
Data container/enum conforming to equatable protocol is a quite standard approach. If it'll be removed accidentally, the test builds failure message is pretty clear what is wrong.

Reply:
Fair enough. Leave it as is then.